src/main/main.ts

```diff
@@ -445,6 +447,45 @@ app.on('window-all-closed', () => {
   app.quit();
 });
 
+/**
+ * Checks for updates from the releases page on github, and, if there is a new version, sends a message to the main window to display a notification
+ */
+const checkAppUpdate = () => {
+  const options = {
+    hostname: 'api.github.com',
+    protocol: 'https:',
+    path: '/repos/aza547/wow-recorder/releases/latest',
+    method: 'GET',
+    headers: {
+      'User-Agent': 'wow-recorder',
+    }
+  }
+
+  const request = net.request(options);
+  request.on('response', (response) => {
+    let data = '';
+
+    response.on('data', (chunk) => {
+      data += chunk;
+    });
+
+    response.on('end', () => {
+      const release = JSON.parse(data);
+      const latestVersion = release.tag_name;
```

Comment:
What is this response doesn't contain the data?

Reply:
Actually I suspect the latter will just never return the event so maybe fine. Still curious about the former.

Reply:
> What is this response doesn't contain the data?
Just added 200 check along with making sure the variables have something in them. If the status 200 page doesn't have the data, it's either a GitHub issue or a release tagging issue.

> What if I start the app with my internet unplugged?
The request will go into request.error and output to logs.